package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "momentum-tokens",
-  "version": "0.0.4",
+  "name": "@momentum-ui/design-tokens",
+  "version": "0.0.13",
```

Comment:
where's this version coming from?

Reply:
That occurred after an `npm install`. I believe this is due to how `package.json` has the version set to `0.0.13` and it does not match the lockfile.